src/main/java/com/quizapp/enterprise/controllers/QuestionController.java

```diff
@@ -22,9 +22,14 @@
 
     @PostMapping("/questions")
     public ResponseEntity<String> addQuestion(@Valid @RequestBody Question questionToAdd){
+        try {
+            questionRepository.save(questionToAdd);
+            return ResponseEntity.ok("Valid Question");
+        }
+        catch {
+            return ResponseEntity.bad("Invalid Question");
```

Comment:
Additional suggestions:
1) Log the error.  Without that, you'll not have very important information when you are trying to figure out why the program crashed.
2) Return an HTTP error code.... something in the 500 series.  That's what they are for.